clj-refactor.el

```diff
@@ -2199,6 +2200,12 @@ the alias in the project."
           (-when-let (long (cljr--aget cljr-magic-require-namespaces short))
             (list short (list long))))))))
 
+(defun cljr--in-sans-ns-keyword-p ()
+  (cljr--sans-ns-keyword?
+   (buffer-substring-no-properties
```

Comment:
`(cider-symbol-at-point)` works here but does not in `(cljr--magic-requires-lookup-alias)`